server/schedulers/shuffle_leader.go

```diff
@@ -118,7 +118,7 @@ func (s *shuffleLeaderScheduler) Schedule(cluster opt.Cluster) []*operator.Opera
 	}
 	op, err := operator.CreateTransferLeaderOperator(ShuffleLeaderType, cluster, region, region.GetLeader().GetId(), targetStore.GetID(), operator.OpAdmin)
 	if err != nil {
-		log.Debug("fail to create shuffle leader operator", zap.Error(err))
+		log.Debug("fail to create shuffle leader operator", zap.Error(errs.ErrCreateOperator.Wrap(err).FastGenByArgs()))
```

Comment:
No need to do it since its level is debug.

Reply:
Just make them compatible, I think there should be no side effects